### ISSUE TITLE
configure github actions

### ISSUE THREAD

**Issue by jesseylin** [flowersteam/adtool_legacy#2]
at minimum,

1. build docs
2. lint and format commits (on any push)

nice to have,

1. run tests
   - autodisc-server-libs
   - postman API check
   - docker health checks

**Comment by jesseylin** [comment]
will consider resolved after #7 and #3, but testing workflows still outstanding and will be moved to new issue #14
